lms/djangoapps/oauth2_handler/tests.py

```diff
@@ -19,14 +20,10 @@
 class BaseTestMixin(ModuleStoreTestCase):
```

Comment:
I realize it's a little out of scope, but is there a chance you could convert this to a [`SharedModuleStoreTestCase`](http://swampcastle.org/2015/07/28/writing-faster-modulestore-tests.html)? It should be pretty straightforward.